core/commands/publish.go

```diff
@@ -91,21 +91,18 @@ Publish an <ipfs-path> to another public key (not implemented):
 			pubValidTime: time.Hour * 24,
 		}
 
-		verif, found, _ := req.Option("resolve").Bool()
-		if found {
-			popts.verifyExists = verif
-		}
-		validtime, found, _ := req.Option("lifetime").String()
-		if found {
-			d, err := time.ParseDuration(validtime)
-			if err != nil {
-				res.SetError(fmt.Errorf("error parsing lifetime option: %s", err), cmds.ErrNormal)
-				return
-			}
+		verif, _, _ := req.Option("resolve").Bool()
+		popts.verifyExists = verif
```

Comment:
Does it make any difference? It is by default true, you can change it only by doing `--verify=false`.